packages/eslint-config-airbnb-base/.eslintrc

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "./index.js",
+  "rules": {
+    // disable requiring trailing commas because it might be nice to revert to
+    // being JSON at some point, and I don't want to make big changes now.
```

Comment:
I'm curious about your thoughts around reverting to being JSON. Any reasons why?